source/includes/_content_access_management.md

```diff
@@ -0,0 +1,283 @@
+
+**Response Codes**:
+
+- `201 CREATED`
+
+
+### Create / Update Access Record Request fields
+
+| Field                                          | Nullable | Type   | Description                                                                                             |
+| ---------------------------------------------- | -------- | ------ | ------------------------------------------------------------------------------------------------------- |
+| contentItemId                                  | false    | string | The content item id for which this access record is responsible for.                                    | 
+| userId                                         | true     | string | If you would like access to be constrained to a specifc user, set this field. Otherwise pass null.      |
+| groupId                                        | false    | string | The group associated to this content item. If userId is null, then all members of the group (with the given role) have access to the content item |
+| role                                           | false    | string | The role this access row applies too. Valid roles are `group_user`, `group_admin`. If you do not want to provide access based on role - pass `any`|
+| canEdit                                        | false    | boolean| Sets whether or not the users this access record applies too are allowed to edit the content. |
+
+## Delete Access Record
+
+> Remove Single Access for a given Content Item Record Request Payload (Request Includes `/{contentItemId}`).
+
+```json
+{
+    "userId": "seattle_user_2",
+    "groupId": "seattle_office",
+    "role": "group_admin"
+}
+```
+
+> Remove All Access Records Associated For A Given Content Item and `groupId` Request Payload (Request Includes `/{contentItemId}`).
+
+```json
+{
+    "userId": null,
+    "groupId": "seattle_office",
+    "role": null
+}
+```
+
+> Remove All Access Records Associated For A Given Content Item Request Payload (Request Includes `/{contentItemId}`).
+
+```json
+{
+    "userId": null,
+    "groupId": null,
+    "role": null
+}
+```
+
+> Remove All Access Records For A Given Repository Matching a Specific `userId` (Request Includes `/{contentItemId}`).
+
+```json
+{
+    "userId": "example_user_id",
+    "groupId": null,
+    "role": null
+}
+```
+
+> Remove All Access Records For A Given Repository Matching a Specific `groupId` (Request Does Not Include `/{contentItemId}`).
+
+```json
+{
+    "userId": null,
+    "groupId": "seattle_office",
+    "role": null
+}
+```
+
+> Remove All Access Records For A Given Repository Matching a Specific `groupId` and `userId` (Request Does Not Include `/{contentItemId}`).
+
+```json
+{
+    "userId": "example_user_id",
+    "groupId": "seattle_office",
+    "role": null
+}
+```
+
+> Remove All Access Records In a Respository (Request Does Not Include `/{contentItemId}`)
+
+```json
+{
+    "userId": null,
+    "groupId": null,
+    "role": null
+}
+```
```

Comment:
these all look a little funny here, does it make sense on the actual page?

Reply:
Yeah.. I think it looks ok. I just wanted to give a number of examples. If we need to change that we can - but I think we can look at it and decide.